river/anomaly/base.py

```diff
@@ -45,6 +48,63 @@ def score_one(self, x: dict) -> float:
 
         """
 
+    def fit(self, X, y=None):
```

Comment:
If I understand correctly, you're using the `_HSChains` internally in the `xStream` class. The `_HSChains` is a batch method, so you need to call `fit`. Even if that's the case, it's not a good idea to add a `fit` method to the `AnomalyDetector` base class.

Reply:
The results have been added in the file benchmarks/xStream_benchmark and random seed has been removed from the class.